public/video-ui/src/components/VideoUpload/Upload.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import {browserHistory} from 'react-router';
+import Icon from '../Icon';
+
+class VideoUpload extends React.Component {
```

Comment:
Filename and component name is different, would it make more sense for this to be `VideoUpload.js`